test/test-function-loading.js

```diff
@@ -0,0 +1,20 @@
+const test = require('tape');
+const path = require('path');
+const { loadFunction } = require('../lib/function-loader.js');
+
+const fixtureDir = path.join(__dirname, 'fixtures');
+
+test('Loads a CJS function', async t => {
+  await loadFunction(path.join(fixtureDir, 'cjs-module', 'index.js'));
```

Comment:
Can you add a check here and in the other two tests similar to this?
```suggestion
  const fn = await loadFunction(path.join(fixtureDir, 'cjs-module', 'index.js'));
  t.equal(typeof fn, 'function');
```

Reply:
will do